src/ConcordiumNetSdk/Wallets/Json/Value.cs

```diff
@@ -0,0 +1,18 @@
+using ConcordiumNetSdk.Types;
+using ConcordiumNetSdk.Crypto;
+
+namespace ConcordiumNetSdk.Wallets.Json;
+
+/// <summary>
+/// Represents (a subset of) a JSON object indexed by the
+/// <c>value</c> field of the browser and genesis wallet
+/// export JSON formats.
+///
+/// Such can be parsed into an instance of this class using
+/// <see cref="Newtonsoft.Json.JsonConvert"/>.
+/// </summary>
+public class Value
```

Comment:
Also this.

Do these need to be separate classes? In that case why are they not records?

Reply:
Opposed to several classes do you mean that they could be nested?

Reply:
Yes. I know this is a Java/C# thing but I find these scattered classes in all the different files rather hard to keep track of.

Reply:
Yea, I agree. I nested as much as possible.